### PR TITLE
feat(scanv4): Initital support for Scanner V4 in bundles 

### PR DESCRIPTION
**Pull request by mclasmeier**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: central/scanner/handler.go [hunk 1]
```diff
@@ -93,6 +107,16 @@ func generateFilesForScannerV1(params *apiparams.Scanner, clusterType storage.Cl
 			"scanner-db-cert.pem": scannerDBCert.CertPEM,
 			"scanner-db-key.pem":  scannerDBCert.KeyPEM,
 			"scanner-db-password": dbPassword,
+
```
Comment (kurlov) [inline central/scanner/handler.go:110]: This comment is about `CommonConfig`. There are scannerV4 configs in image flavor. You might need to pass those scannerV4 image tags as part of `CommonConfig`

Comment (mclasmeier) [reply central/scanner/handler.go:110]: will look into that, thank you!

Comment (mclasmeier) [reply central/scanner/handler.go:110]: Added a ticket here: https://issues.redhat.com/browse/ROX-22036

**Review by kurlov — APPROVED** [review]

File: tests/e2e/run-scanner-v4.bats [hunk 2]
```diff
@@ -121,72 +121,97 @@ teardown() {
...
 @test "Fresh installation of HEAD Helm chart with Scanner v4 enabled" {
+    (
     info "Installing StackRox using HEAD Helm chart with Scanner v4 enabled"
     # shellcheck disable=SC2030,SC2031
     export OUTPUT_FORMAT=helm
     _deploy_stackrox
 
     verify_scannerV2_deployed "stackrox"
     verify_scannerV4_deployed "stackrox"
+    ) >&3 2>&1
```
Comment (kurlov) [inline tests/e2e/run-scanner-v4.bats:175]: Is it for debugging?

**Review by kurlov — COMMENTED** [review]
Changed

**Pull request closed without merging**